### ISSUE TITLE
Failed to initialize ESLint. Have you installed and configured correctly? TypeError: ESLintClass is not a constructor

### ISSUE THREAD

**Issue by mrleblanc101** [nuxt/eslint-config#657]
### Environment

|                      |                                                                                                                                                                                                               |
| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **Operating system** | `macOS 25.3.0`                                                                                                                                                                                                |
| **CPU**              | `Apple M1 Pro (8 cores)`                                                                                                                                                                                      |
| **Node.js version**  | `v22.16.0`                                                                                                                                                                                                    |
| **nuxt/cli version** | `3.33.1`                                                                                                                                                                                                      |
| **Package manager**  | `pnpm@10.19.0`                                                                                                                                                                                                |
| **Nuxt version**     | `4.3.1`                                                                                                                                                                                                       |
| **Nitro version**    | `2.13.1`                                                                                                                                                                                                      |
| **Builder**          | `vite@7.3.1`                                                                                                                                                                                                  |
| **Config**           | `$development`, `app`, `compatibilityDate`, `css`, `devtools`, `eslint`, `fonts`, `future`, `i18n`, `image`, `modules`, `nuxtQuery`, `postcss`, `prismic`, `routeRules`, `runtimeConfig`, `scripts`, `vite`   |
| **Modules**          | `@nuxt/eslint@1.15.2`, `@nuxtjs/prismic@5.2.0`, `@nuxt/fonts@0.14.0`, `@nuxtjs/i18n@10.2.3`, `@nuxt/image@2.0.0`, `@vueuse/nuxt@14.2.1`, `@peterbud/nuxt-query@1.5.2`, `@nuxt/scripts@0.13.2`                 |


### Package

@nuxt/eslint

### Reproduction

.

### Describe the bug

I get this error when starting my project after updating all deps including ESLint to v10.
> ERROR  [uncaughtException] Failed to initialize ESLint. Have you installed and configured correctly? TypeError: ESLintClass is not a constructor

### Additional context

Relates to: https://github.com/ModyQyW/vite-plugin-eslint2/issues/47

### Logs

```shell-script

```

**Comment by AnyFork** [comment]
Same problem !

**Comment by sebbayer** [comment]
It looks like @nuxt/eslint sets `eslintPath` in its configuration to `'eslint/use-at-your-own-risk'` for flat config,
but ESLint 10 removed the ESLint class from that entry point.

For me this workaround in nuxt config fixes the problem:

```
eslint: {
    checker: {
        eslintPath: 'eslint',
   },
},
```

But I think this should be updated in @nuxt/eslint.
